### DISCUSSION TITLE
Slack channel or Discord server

### DISCUSSION THREAD

**Discussion by rlm96** [DevToys-app/DevToys#135] [General]
Hey all,

I think it would be good to have a small Slack channel or Discord server just to improve communication. I'm currently working in a feature and maybe it would help me to solve a couple of doubts faster. Thoughts? 🤔

**Comment by veler** [comment]
Hi,
Thank you for suggesting this. That could help, although there's the GitHub Discussion feature.

In fact, if you join the UWP Community discord server, there's already a DevToys channel there. https://uwpcommunity.com/
We use it to communicate about it with the rest of the UWP Community.

Do you have some ideas in mind of what channels could be useful in a dedicated discord server? I can't think about much stuff except `general discussion`, `updates`, `work in progress`, `dev discussion`, which, to be fair, would pretty much be an echo to what we have on the UWP Community discord + GitHub, with the cost of having to moderate too.

**Reply by rlm96** [reply]
Hey,
Thanks for your answer. I didn´t know about the DevToys channel in the UWP Community Discord server so I think it is perfect to keep that channel and Github Discussions 👌
